pkgs/applications/audio/hushboard/default.nix

```
@@ -0,0 +1,69 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, wrapGAppsHook
+, libpulseaudio
+, gtk3
+, gobject-introspection
+, libappindicator
+, librsvg
+, pycairo
+, pygobject3
+, six
+, xlib
+}:
+
+buildPythonApplication {
+  pname = "hushboard";
+  version = "unstable-2021-03-17";
```

```suggestion
  version = "0.pre+unstable=2021-03-17";
```

(Shameless plug to future https://github.com/NixOS/rfcs/pull/107; feel free to accept or reject it)